File: game-server/src/main/java/org/forfun/mmorpg/game/Modules.java
package org.forfun.mmorpg.game;

public interface Modules {

	// ------------------底层功能支持模块（-128 ~ -1）-----------------
	short BASE = -1;
	short GM = -2;
	short CROSS = -3;

	// ------------------业务功能模块（1~326）---------------------

	/** 玩家 */
	short PLAYER = 1;
	/** 场景 */
	short SCENE = 2;
	/** 活动 */
	short ACTIVITY = 3;
	/** 技能 */
	short SKILL = 4;

}

File: game-server/src/main/java/org/forfun/mmorpg/game/Modules.kt
package org.forfun.mmorpg.game

interface Modules {

    companion object {
        // ------------------底层功能支持模块（-128 ~ -1）-----------------
        const val BASE: Short = -1
        const val GM: Short = -2
        const val CROSS: Short = -3

        // ------------------业务功能模块（1~326）---------------------

        /** 玩家 */
        const val PLAYER: Short = 1
        /** 场景 */
        const val SCENE: Short = 2
        /** 活动 */
        const val ACTIVITY: Short = 3
        /** 技能 */
        const val SKILL: Short = 4
        /** 聊天 */
        const val CHAT: Short = 5
    }
}
